web/themes/custom/nicsdru_corplite_theme/src/scss/2_layouts/layouts-common.scss

```diff
@@ -147,3 +147,46 @@
     word-break: break-word;
   }
 }
+
+.page-node-type-news {
```

Comment:
I'm not entirely sure we need these styles? But we can merge it in anyway and see.